### DISCUSSION TITLE
More field in PRODUCT under PRICE BOOK

### DISCUSSION THREAD

**Discussion by leo719044** [YetiForceCompany/YetiForceCRM#15259] [How to]
how to add more field to the PRODUCT view under PRICE BOOK,  something like "list price". 

I have one standard price list for all agent. but for someone who can apply for the special discount. so I will add those special discount to PRICEBOOK, with comments, valid date and MOQ etc. 

![image](https://user-images.githubusercontent.com/88583995/135855408-1854ccaa-e740-4a1b-b281-c3c95ce7cd1d.png)

**Comment by noelhunter** [comment]
Add a custom field

[https://www.youtube.com/watch?v=l73siRgRtoM](url)

**Reply by leo719044** [reply]
Thanks for the reply.

Unfortunately, not what I want.

I fully understand how to configure basic fields in editing.

But "list price" is different. I can not find it anywhere neither PRODUCT nor PRICEBOOK. and this field only appeared when viewing PRODUCT from PRICEBOOK

**Reply by noelhunter** [reply]
Sorry, that was confusing.  It appears that listprice is a relationship implemented in the code.  If you search for listprice, you can find examples.  You might be able to add another such field by looking at the code and duplicating it for a new field, or you might want to create a custom module.  You can see the code in files such as /modules/PriceBooks/relations/GetPricebookProducts.php
